src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -219,6 +219,7 @@ private void AddOverviewPage(TableOfContentsNode toc)
             if (!string.IsNullOrEmpty(toc.Uid) || !string.IsNullOrEmpty(toc.Href))
             {
                 var overview = new TableOfContentsNode(toc);
+                overview.Expanded = null;
```

Comment:
Setting it to `false` should work, since there is `[JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]`
```suggestion
                overview.Expanded = false;
```

Reply:
Fixed.